lib/AFS/CellCC/Config.pm

```diff
@@ -118,6 +118,8 @@ my @directives = (
     { key => 'vos/command', default => 'vos', },
     { key => 'vos/princ', default => undef, },
     { key => 'vos/keytab', default => "$CONF_DIR/vos.keytab", },
+    { key => 'vos/flags', type => 'HASH', default => {}, },
+    { key => qr:^vos/flags/[^/]+/*:, default => undef, },
```

Comment:
this commit doesn't seem to use the plain "vos/flags" option, and we don't honor flags for arbitrary vos ops, just for release. I think this should just be:

{ key => 'vos/flags/release', type => 'HASH', default -> {}, } ,

Adding configurable flags to all individual vos operations and to vos generically makes sense if you want to do that (but it's not needed right now). But if we want to add generic vos flags to pass for all operations, you can't have "vos/flags" and "vos/flags/release", since "release" looks like a flag to add. So maybe this should be the other way around? That is, make "vos/release/flags", and eventually we could add a generic "vos/flags" later on. If that makes sense to you?

Reply:
Make sense (done).